frontend/integration-tests/tests/kubevirt/non-admin.scenario.ts

```diff
@@ -0,0 +1,173 @@
+/* eslint-disable no-undef */
+import { $, $$, browser, ExpectedConditions as until } from 'protractor';
+import { appHost, testName } from '../../protractor.conf';
+import { nonAdminSecret, nonAdminProvider } from './utils/mocks';
+import { click, createResources } from './utils/utils';
+import { KUBEADMIN_IDP, KUBEADMIN_USERNAME, BRIDGE_HTPASSWD_IDP, BRIDGE_HTPASSWD_USERNAME, BRIDGE_HTPASSWD_USER_EXIST, BRIDGE_HTPASSWD_PASSWORD, BRIDGE_KUBEADMIN_PASSWORD } from './utils/consts';
+import * as loginView from '../../views/login.view';
+import * as crudView from '../../views/crud.view';
+import * as yamlView from '../../views/yaml.view';
+import { createWithWizardLink, createWithYAMLLink, closeWizard } from '../../views/kubevirt/wizard.view';
+import { logout } from '../../views/kubevirt/login.view';
+import { execSync } from 'child_process';
+
+describe('Test nonadmin user behaviour', () => {
+  const nonAdminNS = `${testName}-nonadmin`;
+
+  beforeAll(async() => {
+    if (BRIDGE_HTPASSWD_USER_EXIST === 'false') {
```

Comment:
It can be done at runtime. A better solution is moving the user creation part to the infrastructure so can just use it.

Reply:
I suppose it may be done as part of deployment, feel free to discuss this with Tareq. 
But I'm OK with the way you add the user. Just as long as it doesn't require QE to set env variable based on whether the user is already there or not.

Reply:
Has removed this variable and check it at runtime.